sources/osgShadow/ShadowMap.js

```diff
@@ -919,9 +920,9 @@ ShadowMap.prototype = MACROUTILS.objectLibraryClass( MACROUTILS.objectInherit( S
         this._depthRange[ 2 ] = this._depthRange[ 1 ] - this._depthRange[ 0 ];
         this._depthRange[ 3 ] = 1.0 / ( this._depthRange[ 2 ] );
 
-        this._texture.setViewMatrix( this._viewMatrix, this._lightIndex );
-        this._texture.setProjectionMatrix( this._projectionMatrix, this._lightIndex );
-        this._texture.setDepthRange( this._depthRange, this._lightIndex );
+        this._texture.setViewMatrix( this._lightIndex, this._viewMatrix );
```

Comment:
it's the "lightIndex" in the Array of this_.lights (as index of this array) of the shadowmapAtlas and texture, not the "lightNumber" identifier of light